### ISSUE TITLE
a11y audit: focus indicator color bug fixes

### ISSUE THREAD

**Issue by carylwyatt** [hathitrust/firebird-common#92]
- [x] focus indicator on www/homepage logo in navbar
- [x] secondary buttons with an "active" class have a gray focus ring where the white box shadow should be
- [x] transition on links/buttons
- [x] firefox: box shadow bug
